WordPress/gutenberg
# Empty Theme: Add the `$schema` property in `theme.json` and rename template directories

## What?
This PR makes the following two changes to the Empty Theme:

- Add the `$schema` property in `theme.json`
- Rename directories: `block-templates` to `templates`, `block-template-parts` to `template-parts`

## Why?

This may not be true for all contributors, but I activate Empty Theme when isolating the cause.  I also temporarily add properties when testing the reproducibility of global style-related defects, etc. In those cases, I think it would be useful to have auto-completion and validation.

As for directory names, [they have changed in WordPress 5.9](https://core.trac.wordpress.org/changeset/52247). The Gutenberg plugin supports WordPress versions 5.9 and above, so renaming should not be a problem now.

## Testing Instructions

- Confirm that the template (part) is recognized by the site editor when the Empty Theme is activated.
- Open `theme.json` in the code editor and confirm that auto-completion and validation are performed.
